package.json

```diff
@@ -19,8 +19,8 @@
     "chai-moment": "^0.1.0",
     "coveralls": "^2.11.12",
     "eslint": "^3.2.2",
-    "eslint-config-airbnb": "^10.0.0",
-    "eslint-plugin-import": "^1.8.1",
+    "eslint-config-airbnb": "^13.0.0",
+    "eslint-plugin-import": "^2.1.0",
```

Comment:
Probably nothing to worry about but the version of this in [nhsuk/connecting-to-services/package.json#L69](https://github.com/nhsuk/connecting-to-services/blob/master/package.json#L69) is `2.2.0`